code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -130,7 +135,7 @@
 		if(M == src.summoner || (M in dead_mob_list))
 			M << "<span class='boldannounce'><i>[src]:</i> [input]</span>"
 	src << "<span class='boldannounce'><i>[src]:</i> [input]</span>"
-	log_say("[src.real_name]/[src.key] : [text]")
+	log_say("[src.real_name]/[src.key] : [input]")
```

Comment:
Ha, this is brill
```text``` is the byond var that defaults to the first letter of the atom's name, and it's used for the ascii/text view mode.

rofl